### DISCUSSION TITLE
YYYY-MM-DD when using 'set date'

### DISCUSSION THREAD

**Discussion by hakeem86** [fomantic/Fomantic-UI#1019] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
# Help Wanted

Just want to ask, is YYYY-MM-DD the corect way format the date when using 'set date'
I have tried with diffrent dates and seem to parse correct every time. 

## Problem

https://jsfiddle.net/p4kbnsae/

**Comment by prudho** [comment, marked as answer]
Yup, this date is parsed correctly every time using this format.

**Comment by lubber-de** [comment]
As @prudho said, this format is parsed (as well as other as you found out), but it's not THE format. 
To be on the very safe side, you can always provide a Javascript date object instead of a string.
Closing because your question was answered
